src/routers/createConfigGetter.js

```diff
@@ -98,7 +97,5 @@ export default (
     );
     outputConfig = applyConfig(routeScreenConfig, outputConfig, configOptions);
 
-    validateScreenOptions(outputConfig, route);
```

Comment:
Yeah, I support that. Sadly, that forbids use of `header` which we want to take advantage from here.